subprojects/docs/src/docs/userguide/tutorial/part6_gradle_caching.adoc

```diff
@@ -0,0 +1,148 @@
+// Copyright 2017 the original author or authors.
```

Comment:
Hmmm this is pretty big to add caching to the introduction - but I kinda like it

Reply:
ty